examples/bri-3/src/bri/identity/bpiAccounts/agents/bpiAccounts.agent.ts

```diff
@@ -18,8 +18,17 @@ export class BpiAccountAgent {
 
   public createNewBpiAccount(
     ownerBpiSubjectAccounts: BpiSubjectAccount[],
+    authorizationCondition: string,
```

Comment:
better to define optional if these fields are "placeholders"

Reply:
It is a placeholder, but it is not optional. This PR is created because these fields are specified as requirements in the standard, but were not initially included.